scripts/generate.mjs

```diff
@@ -72,7 +76,7 @@ async function getJsonSchemaToTypescript() {
       `
         set -eu
         cd ./node_modules/json-schema-to-typescript
-        rm -rf dist
+        rm -rf lib
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** json-schema-typescript did not change

```suggestion
        rm -rf dist
```